vendor/visualmetrics.py

```diff
@@ -1490,7 +1494,21 @@ def calculate_contentful_speed_index(progress, directory):
         command = '{0} {1} -canny 2x2+8%+8% -define histogram:unique-colors=true -format %c histogram:info:-'.format(
             image_magick['convert'], current_frame)
         output = subprocess.check_output(command, shell=True)
-        value  = int(output.split()[7].split(':')[0])
+        logging.debug("Output %s" % output)
+
+        # take the last line of the convert call output
+        lines = [line.strip() for line in output.split("\n") if line.strip()]
+        if len(lines) == 0:
+            logging.debug("Could not find the contentfulness value")
+            return -1
+
+        # extract the value from the last line
+        match = matcher.findall(lines[0])
```

Comment:
This would be more clear if it was `lines[-1]`, since you say _last_ line.

Reply:
we are already in the last line, here we pick the first element in the regexp